samples/java/play_2.5/activator

```diff
@@ -288,7 +288,7 @@ declare -a java_args
 declare -a app_commands
```

Comment:
We really should support older Play versions with the same codebase.
Also, just replacing 2.4 to 2.5 here would stop build against Play 2.4 and someday it'll get broken.
It might be better we create the new sample application under samples/java/play_2.5 instead of renaming it.